model/src/test/kotlin/utils/LicenseResolverTest.kt

```diff
@@ -114,7 +114,7 @@ private fun createEmptyOrtResult() =
         )
     )
 
-class LicenseResolverTest : WordSpec() {
+class g : WordSpec() {
```

Comment:
That's probably unintended. But like said, feel free to turn this into a [Behavior Spec](https://github.com/kotest/kotest/blob/master/doc/styles.md#behavior-spec) instead.

Reply:
Done.